andela/apollo-ah-backend
# #165177754 fix confirmation eamil

#### What does this PR do?
This PR changes the link sent to users on registration to a link pointing to the front-end

#### Description of Task to be completed?
- fix confirmation link to point to a front end app

#### How should this be manually tested?
- clone the repo
- install postgress, node js and yarn
- run yarn install to install dependencies
- run yarn start:dev
- create a new user and you should receive an email with a link that points to a site configured on the .env file (WEB_APP)

#### What are the relevant pivotal tracker stories?
`165177754`
